### ISSUE TITLE
New forum post notifications should happen on publish, not creation

### ISSUE THREAD

**Issue by BWPanda** [backdrop-ops/forum.backdropcms.org#104]
When a new forum post is created, a notification is sent to Gitter. That's all well and good, but if Akismet thinks the post might be spam, it un-publishes it. Then Gitter users get an 'access denied' message when clicking the link.

Can we change it so that notifications in Gitter only happen when a post is actually published?

**Comment by stpaultim** [comment]
I agree that this would be good, but hopefully we can solve this https://github.com/backdrop-ops/forum.backdropcms.org/issues/88 first.

**Comment by BWPanda** [comment]
Here's a PR: https://github.com/backdrop-ops/forum.backdropcms.org/pull/116 but I'm not sure if it works properly or not since I can't actually test Akismet (i.e. submit spam and know that Akismet is going to unpublish it)... Any ideas @herbdool?
